apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -63,13 +63,10 @@ loop(Socket, Transport, State) ->
                     InfoBin = dproto_tcp:encode_bucket_info(Info),
                     Transport:send(Socket, InfoBin),
```

Comment:
I tried removing the above lines from my local elvis config and running `rebar3 as lint lint` still passes.  Does the elvis runner on the build system work differently?